api/models/post.js

```diff
@@ -0,0 +1,34 @@
+const Sequelize = require("sequelize");
+const sequelize = require("../db/db.js");
+const User = require("./User.js");
+
+const Post = sequelize.define("posts", {
+  id: {
+    type: Sequelize.INTEGER(11),
```

Comment:
if id name would be **post_id** it will be nice when we put this id in threads table as foregin_key. (it is okay just for readability)